src/components/08-navigation/_nav-primary.njk

```diff
@@ -20,7 +20,7 @@
               <ul class="usa-nav__submenu-list">
                 {%- for child in group -%}
                   <li class="usa-nav__submenu-item">
-                    <a href="{{ child.href }}">{{ child.text }}</a>
+                    <a href="{{ child.href }}" class="{% if child.is_current %} usa-current{% endif %}">{{ child.text }}</a>
```

Comment:
In the PR there was a usa-current link added for testing. Fractal threw it in both subnavs, do we want to include this test code?

Issue:
https://github.com/uswds/uswds/pull/3434#issuecomment-623710243

Reply:
Removing this